python-lib/constants.py

```diff
@@ -0,0 +1,51 @@
+class METRICS_DATASET:
+    """ Class of constants with labels used in the evaluation metrics dataframe of the train recipe """
+
+    TARGET_COLUMN = "target_column"
+    MODEL_COLUMN = "model"
+    AGGREGATED_ROW = "__aggregated__"
+    MODEL_PARAMETERS = "model_params"
+    SESSION = "training_session"
+    TRAINING_TIME = "training_time"
+
+
+class TIMESERIES_KEYS:
+    """ Class of constants with labels for the keys used in the timeseries of the GluonDataset class """
+
+    START = "start"
+    TARGET = "target"
+    TARGET_NAME = "target_name"
+    TIME_COLUMN_NAME = "time_column_name"
+    FEAT_DYNAMIC_REAL = "feat_dynamic_real"
+    FEAT_DYNAMIC_REAL_COLUMNS_NAMES = "feat_dynamic_real_columns_names"
+    IDENTIFIERS = "identifiers"
+
+
+EVALUATION_METRICS_DESCRIPTIONS = {
+    "MSE": "Mean Squared Error",
+    "MASE": "Mean Absolute Scaled Error",
+    "MAPE": "Mean Absolute Percentage Error",
+    "sMAPE": "Symmetric Mean Absolute Percentage Error",
+    "MSIS": "Mean Scaled Interval Score",
+    "RMSE": "Root Mean Square Error",
+    "ND": "Normalized Deviation",
+    "mean_wQuantileLoss": "Mean Weight Quantile Loss",
+}
+
+
+METRICS_COLUMNS_DESCRIPTIONS = {
+    METRICS_DATASET.MODEL_COLUMN: "Model name",
+    METRICS_DATASET.MODEL_PARAMETERS: "Parameters used for training",
+    METRICS_DATASET.SESSION: "Timestamp of training session",
+    METRICS_DATASET.TARGET_COLUMN: "Aggregated and per-time-series metrics",
+    METRICS_DATASET.TRAINING_TIME: "Time to train model for evaluation (in seconds)",
```

Comment:
Since we will change the way a model is trained between after evaluation, I would simplify and not mention evaluation. Also, we need to explain how time is measured, as CPU time can be very different from the Elapsed time.
```suggestion
    METRICS_DATASET.TRAINING_TIME: "Time elapsed during model training (in seconds)",
```